### ISSUE TITLE
Rack v1: pixelRatio is not set correctly on Windows 10, 4K display

### ISSUE THREAD

**Issue by cschol** [VCVRack/Rack#1189]
Rack v1, 540e82dfdd261c51303ea61f15e297e3b3b2545c
Windows 10, 4K display with resolution 3840x2035

There is an issue with the pixel ratio on 4K displays (again). This might be related to what we found last time in Rack issue #89 during `v0.6` development.

Here is how I can make it work:

In `windows.cpp`, change

```
pixelRatio = std::round(pixelRatio);
```
to

```
pixelRatio = 0.5 * std::round(pixelRatio);
```

and everything looks normal (and works, i.e. mouse coordinates etc).

`pixelRatio` reports back as value 2 on my machine. Per the code above it would have to be 1?

**Comment by AndrewBelt** [comment]
The reason this functionality "broke" is because it was broken on many people's machines before, so I threw away the custom DPI-detecting code, and I'll rewrite it. I'll buy a high-DPI Windows 10 laptop and test across multiple machines, and then release the solution for public testing.

**Comment by cschol** [comment]
Enabling these two commented out lines in `window.cpp` (l.381) makes it work also:

```
// nvgReset(vg);
// nvgScale(vg, pixelRatio, pixelRatio);
```

Is that what you mean by `DPI detecting code`?

**Comment by cschol** [comment]
One cosmetic issue when you enable the two lines mentioned above: the menu names are cut off.

![image](https://user-images.githubusercontent.com/624427/52180891-b4a1c380-27a8-11e9-9d23-728c114efdf0.png)

But everything else I tested works fine, namely the screen scaling and mouse coordinates on a 4K screen works like it did on `v0.6`.

**Comment by lnikj** [comment]
The menu name issue also happens on Retina (6700K; 395X; 10.13.6; 5K screen). If I move Rack to an HD monitor the overflow is gone, but move it back to the 5K screen and the overflow is back.

**Comment by cschol** [comment]
I created a new issue for the menu names overflow. The original issue here is the fact that the scaling is not applied per the pixelRatio reported from the OS. I updated the issue name also.

@AndrewBelt Are the two lines in this [comment](https://github.com/VCVRack/Rack/issues/1189#issuecomment-460072288) meant to be commented out?

**Comment by AndrewBelt** [comment]
Acquired a Windows 10 laptop. Fixed in 26247a0c97822a710cd027c7f00f04367a73b765. Your `pixelRatio` should be 2. Solution was to reenable scaling in the NanoVG context.

**Comment by cschol** [comment]
Perfect. Thank you! And congrats on the new laptop. :)
